brainstormforce/astra
# fix: Footer widget options missing

### Description
fix: Footer widget options missing and panel too.

### Screenshots
- https://a.cl.ly/P8u5xY4W

### Types of changes
- Configs were missing so managed the configs with array operations.

### How has this been tested?
- Check all widget options.
- Check console errors.

### Checklist:
